Cython/Utility/TypeConversion.c

```diff
@@ -88,7 +88,28 @@ static CYTHON_INLINE PyObject* __Pyx_PyUnicode_FromString(const char*);
 #define __Pyx_PyUnicode_FromOrdinal(o)       PyUnicode_FromOrdinal((int)o)
 #define __Pyx_PyUnicode_AsUnicode            PyUnicode_AsUnicode
 
-#define __Pyx_NewRef(obj) (Py_INCREF(obj), obj)
+#if CYTHON_COMPILING_IN_CPYTHON && PY_VERSION_HEX >= 0x030a00b1
```

Comment:
Don't change this but we only worry about getting the exact version hex for pre-releases versions of older Python versions - we aren't supporting 3.10 betas at this stage so I'd have used `0x030A0000` and called it good enough (and saved a bit of time looking up exactly when it was introduced).

Reply:
Totally agree. I'll change it since this was probably a copy paste from somewhere else and I didn't look too closely to see what exact prerelease introduced this.